spec.html

```diff
@@ -7201,18 +7201,18 @@ <h1>
       </emu-alg>
     </emu-clause>
 
-    <emu-clause id="sec-disposeresources-disposable-completion-errors" type="abstract operation">
+    <emu-clause id="sec-disposeresources" type="abstract operation">
       <h1>
         DisposeResources (
-          _disposable_: an object with a [[DisposableResourceStack]] internal slot or *undefined*,
+          _disposable_: an Object that has a [[DisposableResourceStack]] internal slot or *undefined*,
```

Comment:
As above, it's possible neither version is correct. This can either be an ECMAScript Object or a declarative Environment Record.